datalab/datalab_session/utils/file_utils.py

```diff
@@ -101,16 +101,20 @@ def create_tif(fits_paths: np.ndarray, tif_path, color=False, zmin=None, zmax=No
   max_height, max_width = max(get_fits_dimensions(fp) for fp in fits_paths)
   fits_to_img(fits_paths, tif_path, TIFF_EXTENSION, width=max_width, height=max_height, color=color, zmin=zmin, zmax=zmax)
 
-def crop_arrays(array_list: list):
+def crop_arrays(array_list: list, flatten=False):
```

Comment:
Changed to optionally flatten the data in place with ravel() when cropping, and also return the cropped data size with the data arrays since it might be useful later.